src/interfaces/coral_web/src/components/Conversation/MessagingContainer.tsx

```diff
@@ -166,64 +151,47 @@ const Content: React.FC<Props> = (props) => {
   );
 };
 
-type MessagesProps = Props & { welcomeMessageEnabled: boolean };
+type MessagesProps = Props & { startOptionsEnabled: boolean };
 /**
  * This component is in charge of rendering the messages.
  */
 const Messages = forwardRef<HTMLDivElement, MessagesProps>(function MessagesInternal(
-  { welcomeMessageEnabled, onRetry, messages, streamingMessage, startOption, onStartOptionChange },
+  { startOptionsEnabled, onRetry, messages, streamingMessage, onPromptSelected },
   ref
 ) {
-  const lastMessage = messages[messages.length - 1];
-
   return (
     <div className="mt-auto flex flex-col gap-y-4 px-4 py-6 md:gap-y-6" ref={ref}>
-      <WelcomeMessage
-        show={
-          welcomeMessageEnabled && messages.filter((m) => !isNotificationMessage(m)).length === 0
-        }
-        startOption={startOption}
-        onStartOptionChange={onStartOptionChange}
-      />
+      {startOptionsEnabled && (
```

Comment:
ahh i think we're not missing some logic to hide this after a message has been sent:
<img width="1512" alt="image" src="https://github.com/cohere-ai/cohere-toolkit/assets/5898755/5744609d-b721-4f74-b598-d570b616217a">

also can we centre this on the page like we have it in Coral?
|toolkit|coral|
|-|-|
|<img width="1512" alt="image" src="https://github.com/cohere-ai/cohere-toolkit/assets/5898755/c90ec274-216a-40e0-911c-f4b0b1372821">|<img width="1261" alt="image" src="https://github.com/cohere-ai/cohere-toolkit/assets/5898755/8a98d766-ac8f-4019-a6ff-974b202ace65">|